### ISSUE TITLE
Should the multi-gpu learning rate be adapted to the number of gpu?

### ISSUE THREAD

**Issue by zhou-frank** [junyanz/pytorch-CycleGAN-and-pix2pix#779]
should I scale the multi-gpu lr to batchsize * single-gpu lr?
e.g. pix2pix model default lr = 0.0002(1 batchsize 1 gpu). When I use 16 batchsize and four gpu(4 image per gpu),I think we should set lr = 0.0002 * 16 = 0.0032.
Meanwhile, using WarmupScheduler to avoid gradient exploding problem.
thanks for any help.

**Comment by junyanz** [comment]
You can use a higher learning rate with larger batch size. More recent [work](https://openreview.net/pdf?id=B1Yy1BxCZ) to read.
